src/components/ProgramSelectors/SelectorItem/styles.css

```diff
@@ -0,0 +1,51 @@
+.selector-item {
+    border-bottom: 1px #e8e8e8 solid;
+
+    .heading {
+        display: flex;
+        align-items: center;
+        background: #fafafa;
+        padding: 20px;
+    
+        .collapse-icon {
+            width: 10%;
+            font-size: 14px;
+
+            &:hover {
+                cursor: pointer;
+            }
+        }
+    
+        .icon {
+            width: 12%;
+            color: var(--color-primary);
+            font-size: 18px;
+        }
+
+        .minimized-icon {
+            color: var(--color-primary);
+            font-size: 18px;
+        }
+    
+        .name {
+            text-transform: uppercase;
+            letter-spacing: 2px;
+            font-size: 14px;
+            font-weight: bold;
+        }
+    }
+
+    .search-icon {
+        margin-left: 25px;
+        font-size: 18px;
+    }
+
+    .centered {
+        justify-content: center;
+    }
+    
+    .tree-input {
+        background-color: #eeeeee;
+    }
+}
+
```

Comment:
Use consistent variables here
Do not worry about making it look similar to mockup.
Lets make it consistent first and we can ask the designer to make it consistent.